spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -19,4 +19,8 @@ A new method, `getGroupId()`, has been added to the `ConsumerSeekCallback` inter
 This method allows for more selective seek operations by targeting only the desired consumer group.
 For more details, see xref:kafka/seek.adoc#seek[Seek API Docs].
 
+=== ConcurrentContainerStoppedEvent
+
+A new event `ConcurrentContainerStoppedEvent` will be published once the parent container i.e ConcurentContainerMessageListenerContainer is stopped.
```

Comment:
The `ConcurentContainerMessageListenerContainer` must be present as a code snippet.
I also don't find that this sentence is correct.
May be something like:
```
The `ConcurentContainerMessageListenerContainer` emits now a `ConcurrentContainerStoppedEvent` when all of its child containers are stopped.
```

`i.e.` has to be with the period.
And better to not use it at all, but rather the whole phrase you wanted to express here.
We also may go on where exactly comma is supposed to be around this `i.e.`: https://www.bachelorprint.com/commas/comma-before-or-after-ie/

😄